docs/sources/_index.md

```diff
@@ -6,137 +6,84 @@ aliases:
 cascade:
   TEMPO_VERSION: latest
   PYROSCOPE_VERSION: latest
-description: Guides, installation, and feature documentation
+description: Landing page for Grafana OSS and Enterprise documentation
```

Comment:
I'm not sure if this description works as a call to action for users to click on this page's link if it comes up in social media or search engines. 

Including the product name is a good improvement though.